config.lua.dist

```diff
@@ -20,10 +20,12 @@ expFromPlayersLevelRange = 75
 -- NOTE: maxPlayers set to 0 means no limit
 -- NOTE: allowWalkthrough is only applicable to players
 ip = "127.0.0.1"
+luaApiIp = "127.0.0.1"
 bindOnlyGlobalAddress = false
 loginProtocolPort = 7171
 gameProtocolPort = 7172
 statusProtocolPort = 7171
+luaApiProtocolPort = 7179
```

Comment:
Why not continue the port range normally, so 7173? Kinda random to just jump from 7172 to 7179 as default value

Reply:
there was no specific reason for 7179 but there was a reason to not chose 7173 which was that quite some servers use cast systems and some of them rely on port 7173